frontend/src/pages/trucks/components/add-truck-form/libs/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { type TruckFormModel } from '~/packages/trucks/libs/types/types.js';
```

Comment:
I'm not sure if it's ok to have constants.ts or maybe we should follow the pattern like we do for types/enums/helpers/... (a separate folder called "consts")